Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -400,21 +403,33 @@ - (void)writeAvatarToDisk:(UIImage *)avatar
     });
 }
 
-// TODO: The exact API & encryption scheme for avatars is not yet settled.
-- (void)uploadAvatarToService:(NSData *)data
-                     fileName:(NSString *)fileName
+- (NSData *)encryptedAvatarData:(NSData *)plainTextData outDigest:(NSData **)outDigest
```

Comment:
I know this code is going away, but I'll mention it anyway.

First, I'm not a big fan of the ** "writable param to return secondary result" pattern.

Secondly, when using that pattern, I like to make sure that the first thing that the method/function does is nil that param so if we ever exit early, that writable param has a predictable value.